src/SolverInterface/lpqp_to_conic.jl

```diff
@@ -239,6 +248,27 @@ for f in [:optimize!, :status, :getsolution, :getobjval, :getvartype]
     @eval $f(model::LPQPtoConicBridge) = $f(model.lpqpmodel)
 end
 
+function getdual(model::LPQPtoConicBridge)
+    if model.Asoc !== nothing || model.Arsoc !== nothing
```

Comment:
Normally if `Asoc` or `Arsoc` are nothing, then `socconstr_idx`, and `rsocconstr_idx` are empty.